dashboard/src/components/EncryptionKey.js

```diff
@@ -108,7 +108,9 @@ const EncryptionKey = ({ isMain }) => {
       const encryptedConsultations = await Promise.all(
         consultations.map(prepareConsultationForEncryption(organisation.consultations)).map(encryptItem)
       );
-      const encryptedTreatments = await Promise.all(treatments.map(prepareTreatmentForEncryption).map(encryptItem));
+      const encryptedTreatments = await Promise.all(
+        treatments.map((t) => prepareTreatmentForEncryption({ ...t, user: t.user || user._id })).map(encryptItem)
+      );
```

Comment:
déjà traité par la PR #1310 

```suggestion
      const encryptedTreatments = await Promise.all(treatments.map(prepareTreatmentForEncryption).map(encryptItem));
```